packages/extension-sample/package.json

```diff
@@ -1,24 +1,25 @@
 {
-    "name": "@salesforce/extension-sample",
-    "version": "1.0.0-dev",
-    "private": true,
-    "scripts": {
-    },
-    "peerDependencies": {
-      "@loadable/component": "^5.15.3",
-      "react": "^18.2.0",
-      "react-dom": "^18.2.0"
-    },
-    "devDependencies": {
-      "@loadable/component": "^5.15.3",
-      "@salesforce/pwa-kit-react-sdk": "4.0.0-dev",
-      "@types/react": "~18.2.0",
-      "@types/react-dom": "~18.2.1",
-      "react": "^18.2.0",
-      "react-dom": "^18.2.0"
-    },
-    "engines": {
-      "node": "^16.11.0 || ^18.0.0 || ^20.0.0",
-      "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
-    }
-  }
\ No newline at end of file
+  "name": "@salesforce/extension-sample",
```

Comment:
Does our linter care about package.json files being 2-4 spaces? Either way, you might want to fix this so that only the intended changes for your PR are reflected in the diff.

Reply:
The extension-sample is only for testing. I plan to remove it before merging the PR.